src/app/(nextjs_migration)/(guest)/page.tsx

```diff
@@ -0,0 +1,152 @@
+            <h3>{l10n.getString("prevent-fraud")}</h3>
+            <p>{l10n.getString("prevent-fraud-keep-info")}</p>
+          </li>
+          <li>
+            <h3>{l10n.getString("get-alerts")}</h3>
+            <p>{l10n.getString("get-alerts-find-out")}</p>
+          </li>
+        </ul>
+      </section>
+
+      <section className="how-it-works" data-enter-transition>
+        <h2>{l10n.getString("how-it-works")}</h2>
+        <ol>
+          <li>
+            <Image alt="" src={LaptopImage} />
+            <h3>{l10n.getString("check-for-breaches")}</h3>
+            <p>{l10n.getString("check-for-breaches-we-search")}</p>
+          </li>
+          <li>
+            <Image alt="" src={LockImage} />
+            <h3>{l10n.getString("protect-accounts")}</h3>
+            <p>{l10n.getString("protect-accounts-clear-steps")}</p>
+          </li>
+          <li>
+            <Image alt="" src={MailImage} />
+            <h3>{l10n.getString("alerts-for-breaches")}</h3>
+            <p>{l10n.getString("alerts-for-breaches-monitor-new")}</p>
+          </li>
+        </ol>
+      </section>
+
+      <section className="safe-with-us" data-enter-transition>
+        <div>
+          <h2>{l10n.getString("safe-with-us")}</h2>
+          <p>{l10n.getString("parent-company")}</p>
+          <p>{l10n.getString("our-mission")}</p>
+          <p>
+            <a href="https://www.mozilla.org/mission/" target="_blank">
+              {l10n.getString("learn-more-mission")}
+            </a>
+          </p>
+        </div>
+        <figure>
+          <Image alt="" src={NaturePhoneImage} />
+        </figure>
+      </section>
+
+      <section className="top-questions-about-monitor" data-enter-transition>
+        <div>
+          <h2>{l10n.getString("top-questions-about-monitor")}</h2>
+          <a
+            href="https://support.mozilla.org/kb/firefox-monitor-faq"
+            target="_blank"
+          >
+            {l10n.getString("see-all-faq")}
+          </a>
+        </div>
+        <div>
+          <details>
+            <summary>{l10n.getString("what-is-breach")}</summary>
+            <p>{l10n.getString("when-info-exposed")}</p>
+          </details>
+          <details>
+            <summary>{l10n.getString("what-do-i-do")}</summary>
+            <p>{l10n.getString("visit-monitor-to-learn")}</p>
+          </details>
+          <details>
+            <summary>{l10n.getString("what-gets-exposed")}</summary>
+            <p>{l10n.getString("depends-on-hackers")}</p>
+          </details>
+        </div>
+      </section>
+
+      <section className="see-if-data-breach" data-enter-transition>
+        <h2>{l10n.getString("see-if-data-breach")}</h2>
+        <a
+          className="button primary"
+          data-cta-id="landing-2"
+          href="/user/breaches"
+        >
+          {l10n.getString("get-started")}
+        </a>
+        <p className="hibp-attribution">
+          {l10n.getString("hibp-footer-attribution")}
```

Comment:
The `<b>` here is a plain string rather than a DOM element:

![image](https://github.com/mozilla/blurts-server/assets/4251/bf3b97f0-658f-4104-83e6-4441066b1f8e)

Reply:
Fixed in [2971d96](https://github.com/mozilla/blurts-server/pull/3050/commits/2971d9670154eb777a08bb920f84026d98b9e638).